content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

```diff
@@ -781,10 +781,10 @@ crontab "my-new-cron-object" created
 
 ### Defaulting
 
-{{< feature-state state="beta" for_kubernetes_version="1.16" >}}
+{{< feature-state state="stable" for_kubernetes_version="1.17" >}}
```

Comment:
The preview shows the version at 1.16 -- seems odd. Perhaps a problem with netlify.